media-libs/libshumate/libshumate-1.0.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson vala
+
+DESCRIPTION="Shumate is a GTK toolkit providing widgets for embedded maps"
+HOMEPAGE="https://wiki.gnome.org/Projects/libshumate"
+
+SLOT="0"
+LICENSE="LGPL-2.1+"
+KEYWORDS="~amd64"
+REQUIRED_USE="gtk-doc? ( introspection )"
+
+IUSE="introspection gtk-doc vala" # vector-renderer is still experimental, maybe put in at a later release
+
+RDEPEND="
+	>=dev-libs/glib-2.68.0:2
+	>=x11-libs/cairo-1.4
+	>=dev-db/sqlite-1.12:3
+	>=gui-libs/gtk-4:4
+	>=net-libs/libsoup3-3:3
+	>=dev-libs/json-glib-1.6.0[introspection?]
+	introspection? ( >=dev-libs/gobject-introspection-0.6.3:= )
+"
+# vector-renderer? ( dev-libs/protobuf-c )
+
+DEPEND="${RDEPEND}"
+BDEPEND="
+	gtk-doc? ( >=dev-util/gi-docgen-2021.1 )
+	vala? ( $(vala_depend) )
+"
+
+src_configure() {
+	local emesonargs=(
+		$(meson_use introspection gir)
+		$(meson_use vala vapi)
+		$(meson_use gtk-doc gtk_doc)
+		-Ddemos=false # only built, not installed
+		# $(meson_use vector-renderer vector_renderer)
+		-Dvector_renderer=false
+		-Dlibsoup3=true
+	)
+	meson_src_configure
+}
+
```

Comment:
Okay, tests fail with `(/var/tmp/portage/media-libs/libshumate-1.0.1/work/libshumate-1.0.1-build/tests/memory-cache:53): Gtk-WARNING **: 12:44:53.152: cannot open display: `. Looks like they need to be run under `virtx`.